database/migrations/2018_09_09_111023_create_dev_rant_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateDevRantTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('devRant_community', function (Blueprint $table) {
```

Comment:
Database standards are always using snakecase, fully lowercase

Reply:
fixed it

Reply:
I'm not 100% sure but I think it should be fully lowercase. This is mainly because Laravel uses does some magic with models and the database. This might cause issues later on.
```suggestion
        Schema::create('devrant_community', function (Blueprint $table) {
```